sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpConnectionScope.cs

```diff
@@ -583,10 +583,10 @@ public override void Dispose()
             {
                 StopTrackingLinkAsActive(link);
 
-                // Aborting the session will perform any necessary cleanup of
+                // Closing the session will perform any necessary cleanup of
```

Comment:
There was some reason we used Abort, but I can't recall what it was.    I just double-checked the T1 code and both SB and EH are calling `SafeClose` on the session there.

Should we be using `SafeClose` in Event Hubs too?

Reply:
I only checked the `MessageReceiver` but that's calling `SafeClose`.